tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -423,14 +424,22 @@ func newConfigHotRegionCommand() *cobra.Command {
 		Short: "balance-hot-region-scheduler config",
 		Run:   listSchedulerConfigCommandFunc,
 	}
+
+	// Deprecated: 'list' command will be deprecated in future version, use 'show' command instead.
 	c.AddCommand(&cobra.Command{
 		Use:   "list",
+		Short: "list the config item (will be deprecated in feature version, use show command instead)",
+		Run:   listSchedulerConfigCommandFunc,
+	}, &cobra.Command{
```

Comment:
how about use ‘Aliases’ to replase it.

Reply:
Below is the comparison of `-h` output between command and alias. As you can see, alias will not be printed in help section which could cause confusion.

Define `show` as alias of `list`
```
Available Commands:
  list        list the config item (Deprecated: will be deprecated in feature version, use show command instead)
  set         set the config item
```

Define `show` as command:
```
Available Commands:
  list        list the config item (Deprecated: will be deprecated in feature version, use show command instead)
  set         set the config item
  show        list the config item
```